docs/_jan-milestone_.md

```diff
@@ -0,0 +1,57 @@
+# Jan milestone
+
+## Overview
+
+- Take some time to:
+  - fix bugs, improve developer experience, fix docs
+  - triage open issues across all the repos, not just `loopback-next`
+- Migration work continues to be one of the focus
+- Focus on issues/PRs for loopback-connector-mongodb
+
+## Goals
+
+- [ ] Migration guide between LB3 and LB4 MVP #453
+
+  - [ ] [3] How to migrate datasources from LB3 to LB4 #3946
+  - [ ] [3] How to migrate boot scripts #3957
+
+- [ ] Improve developer experience
+
+  - [ ] [3] Expose isActive() method in Transaction interface #3471
+  - [ ] [3] Missing model.ts after running lb4 openapi command #4135
+  - [ ] [2] Model strict: false is ignored with Postgre SQL repository #4042
+
```

Comment:
IIRC, we wanted to also add one or two tasks for defining models at runtime, see https://github.com/strongloop/loopback-next/pull/4116/files#r357149740

```markdown
#### Create models & REST APIs dynamically at runtime

- Dynamic binding/rebinding of controllers after app start #433
- How to build models, repositories and controllers dynamically at runtime #4296
```

I am proposing to commit the first task for January and the second for February. Thoughts?

Reply:
I've put #433 to committed stories in Jan, and #4296 as stretch goal.